app/_data/docs_nav_kuma_2.1.x.yml

```diff
@@ -95,28 +96,39 @@ items:
             url: "/quickstart/universal/#next-steps"
       - text: Standalone deployment
         url: /deployments/stand-alone/
+        src: /production/deployment/stand-alone/
         items:
           - text: About
             url: "/deployments/stand-alone/#standalone-deployment"
+            src: /production/deployment/stand-alone/
           - text: Limitations
             url: "/deployments/stand-alone/#limitations"
+            src: /production/deployment/stand-alone/
           - text: Usage
             url: "/deployments/stand-alone/#usage"
+            src: /production/cp-deployment/stand-alone/
           - text: Failure modes
             url: "/deployments/stand-alone/#failure-modes"
+            src: /production/cp-deployment/stand-alone/
       - text: Multi-zone deployment
         url: /deployments/multi-zone/
+        src: /production/deployment/multi-zone/
         items:
           - text: About
             url: "/deployments/multi-zone/#multi-zone-deployment"
+            src: /production/deployment/multi-zone/
           - text: Usage
             url: "/deployments/multi-zone/#usage"
+            src: /production/cp-deployment/multi-zone/
```

Comment:
@fabianrbz I think this one is causing the Netlify build to fail and I'm not sure why since it shouldn't be using the URL as a filename. Here's the error from the Netlify logs:

```
Error message
11:53:10 AM:   Deploy did not succeed: Invalid filename 'docs/2.1.x/deployments/multi-zone/#usage/index.html'. Deployed filenames cannot contain # or ? characters
```

Reply:
Looks like this is a weird edge case for the generator.
The problem is how the generator interprets the different items. It works like this (simplified version):
* if the item has a `src` it will generate the page based on that file and use the `url` as the file name of the page
* if it doesn't have a `src` but it does have a `url`, it uses the `url` as the source and generates the page unless the URL has a `hash` fragment. In that case, it assumes that the link is just a reference to a section of the page, so there's no need to generate anything, it should be generated by a different item.
Like in [this](https://github.com/Kong/docs.konghq.com/blob/main/app/_data/docs_nav_gateway_3.2.x.yml#L564-L585) case, the first item generates the page, the others skip the page generation and point to sections of the page.

One problem is that you are using a `hash` fragment AND a `src` at the same time, so the generator will generate a page for that item, and the `hash` ends up being a part of the file name of the page. The other problem is that you are using two different files as the source for the same URL, it should be one file for each URL.

Fortunately, there's a simple fix.
[Here](https://github.com/kumahq/kuma-website/blob/7cd0e2deb25df964fdebdbcf65f7699987440f25/app/_data/docs_nav_kuma_2.1.x.yml#L113-L131) the top level item can generate the page based on `src: /production/cp-deployment/multi-zone/` and the subitems can drop the `src`. This tells the generator to generate the page based on that `src` for the top-level item, and skip the generation for the subitems. They will be just links with a `hash` fragment that link to sections of the page.